client/webserver/api.go

```diff
@@ -733,6 +733,25 @@ func (s *WebServer) apiWalletSettings(w http.ResponseWriter, r *http.Request) {
 	}, s.indent)
 }
 
+// apiToggleWalletStatus updates the wallet's status.
+func (s *WebServer) apiToggleWalletStatus(w http.ResponseWriter, r *http.Request) {
+	form := new(walletStatusForm)
+	if !readPost(w, r, form) {
+		return
+	}
+	err := s.core.ToggleWalletStatus(form.AssetID, form.Disable, form.Force)
```

Comment:
Since we're not allowing forced disables through the UI, it's clearer, imo, to hard code the value `false` here and drop the `Force bool` field. Otherwise, someone has to go look in javascript to see that there is no real option.

Reply:
> Since we're not allowing forced disables through the UI, it's clearer, imo, to hard code the value false here and drop the Force bool field.

We allow force disable just like `Rescan` && `Recovery`. Unless we no longer want force disable, we can remove the `Force bool` Field and parameter.

Reply:
Well, what's the use case for force disable?  Rescan and Recovery have clear use cases, when something is broken while you have an active swap, and a rescan or recover if the only recourse.  But why would disabling the wallet be useful with active orders?

Reply:
A` Force` shutdown could be useful if a market order has been placed and there is no match yet(also, market orders do not have a cancel UI like limit orders). However, the current behaviour I've noticed is that a limit order will return an `activeOrder` error when trying to disable a wallet but a market order(without a match) does not.

Reply:
> A` Force` shutdown could be useful if a market order has been placed and there is no match yet

Do you mean when the market order is still in epoch status?  That's definitely a time NOT to disable the wallet.  Facilitating that would mean giving the user a tool to break the rules.  Not a good use case.  If they are set on doing that, they're going to have to kill dexc.

> (market orders do not have a cancel UI like limit orders, I think).

There is no cancelling an epoch status market order because such an order is never booked.

> However, the current behaviour I've noticed is that a booked limit order(without a match) will return an `activeOrder` error when trying to disable a wallet but a market order(without a match) does not (is this an expected behaviour?).

An epoch status market order?  Or a market order that went through the epoch matching cycle and did not match with a maker trade (OrderStatusExecuted)?
The first should absolutely be considered and active order, the latter not.

Reply:
> There is no cancelling an epoch status market order because such an order is never booked.

This is clear. My order could not have completed the epoch matching cycle. I Will get rid of the force disable. I thought you meant we could provide special handling for `acvtiveOrders` error ([comment](https://github.com/decred/dcrdex/pull/1804#discussion_r958823665))

Reply:
Oh, yeah, sorry for the misleading you there.  I just meant we could recognize the code and display a more clear, and perhaps translated, error than we'd get from the go code's `error`